modules/rules_abs/0.1.0/MODULE.bazel

```diff
@@ -0,0 +1,82 @@
+module(
+    name = "rules_abs",
+    version = "0.1.0",
+    compatibility_level = 0,
+)
+
+# ✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂
+# only dev_dependencies below this line - rules_abs is lean
+# ✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂✂
+
+bazel_dep(
+    name = "tweag-credential-helper",
+    version = "0.0.5",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.7.1",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "rules_bazel_integration_test",
+    version = "0.32.0",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "rules_java",
+    version = "8.11.0",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "rules_proto",
+    version = "7.1.0",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "rules_testing",
+    version = "0.8.0",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "stardoc",
+    version = "0.8.0",
+    dev_dependency = True,
+)
+bazel_dep(
+    name = "toolchains_protoc",
+    version = "0.3.7",
+    dev_dependency = True,
+)
+
+protoc = use_extension(
+    "@toolchains_protoc//protoc:extensions.bzl",
+    "protoc",
+    dev_dependency = True,
+)
+protoc.toolchain(
+    google_protobuf = "com_google_protobuf",
+    version = "v27.0",
+)
+
+http_jar = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_jar")
+
+http_jar(
+    name = "protobuf-java",
+    sha256 = "9072e60fe66cff5d6c0f11a1df21d8f3e4b29b5ee782b45c3fc75f59fbe2b839",
+    urls = ["https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/4.27.0/protobuf-java-4.27.0.jar"],
```

Comment:
drive-by comment: this `http_jar` can also be marked a `dev_dependency`

Reply:
done, thanks!